src/core/defaults.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Sets the core defaults
+ */
+export const name = "core/defaults";
+import { rule as checkInternalSlots } from "./core/linter-rules/check-internal-slots";
+import { rule as checkPunctuation } from "./core/linter-rules/check-punctuation";
+import { definitionMap } from "./core/dfn-map";
+import linter from "./core/linter";
+import { rule as localRefsExist } from "./core/linter-rules/local-refs-exist";
+import { rule as noHeadinglessSectionsRule } from "./core/linter-rules/no-headingless-sections";
+import { rule as noHttpPropsRule } from "./core/linter-rules/no-http-props";
+
+linter.register(
+  noHttpPropsRule,
+  noHeadinglessSectionsRule,
+  checkPunctuation,
+  localRefsExist,
+  checkInternalSlots
+);
+
+const coreDefaults = {
+  lint: {
+    "no-headingless-sections": true,
+    "no-http-props": true,
+    "check-punctuation": false,
+    "local-refs-exist": true,
+    "check-internal-slots": false,
+  },
+  pluralize: true,
+  highlightVars: true,
+  license: "CC-BY",
+  addSectionLinks: true,
+};
+
+export function run(conf) {
+  if (conf.specStatus === "unofficial") return;
```

Comment:
this line can go, as this will apply to all document, including "unofficial" ones.